### DISCUSSION TITLE
Display archival reason as an obvious and large-font one-liner

### DISCUSSION THREAD

**Discussion by es-fabricemarie** [community/community#67401] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

When administrators archive a repo, they get reminded that they should modify the README.md to give some explanation or a new URL to a superseding project.

However, this forces administrators to push one more change to their repo
- this may require additional approvals
- this may trigger unwanted actions
- etc..

I propose that during archival confirmation dialog, GH asks the repo administrator for a one-liner text explaining the users the reason for this archival. When users browse the repo, they get served the repo landing page as usual with a preceding warning stanza containing the one-liner given.
- it needs to be visually obvious that the repo has been archived (already the case)
- it needs to mention who performed the archival
- the one liner explaination must allow URLs
- the one liner explanation should not live in the repo itself (so admins do not have to perform one last merge).

Today on repo archival, the following built-in warning will be displayed on a yellow banner at the top of the page:

```
This repository has been archived by the owner on Sep 18, 2022. It is now read-only.
```

I propose that it be changed to something like:

```
Repository archived by <username> on Sep 18, 2022. It is now read-only.

Project ABC is obsolete and has now been superseded by Project Awesome. The archive is kept online for sentimental and educational value.
```

**Comment by brooksdavis** [comment]
Another use for this would be to redirect people to the correct repo (e.g., if the contents have been incorporated into another repo or a product has been replaced.)
